src/node/destination.rs

```diff
@@ -104,7 +104,7 @@ impl AudioDestinationNode {
     }
     /// The maximum number of channels that the channelCount attribute can be set to (the max
     /// number of channels that the hardware is capable of supporting).
-    pub fn max_channels_count(&self) -> usize {
+    pub fn max_channel_count(&self) -> usize {
```

Comment:
Please also add a [link](https://www.w3.org/TR/webaudio/#AudioDestinationNode-attributes) to the spec as discussed.